Sources/SpeziChat/ChatView+SpeechButton.swift

```diff
@@ -31,6 +33,34 @@ private struct ChatViewSpeechButtonModifier: ViewModifier {
                                 .accessibilityLabel(Text("Text to speech is disabled, press to enable text to speech.", bundle: .module))
                         }
                     }
+                    .simultaneousGesture(LongPressGesture().onEnded { _ in
+                        isVoiceSelectionSheetPresented.toggle()
+                        muted = false
+                    })
```

Comment:
Wondering why mute is set to false here, might be a bit confusing if that state is changed as well?

Formatting might also be nicer if we try to break this up into multiple lines:
```suggestion
                    .simultaneousGesture(
                        LongPressGesture()
                            .onEnded { _ in
                                isVoiceSelectionSheetPresented.toggle()
                                muted = false
                            }
                    )
```